react/Viewer/Toolbar.jsx

```diff
@@ -51,16 +51,21 @@ const Toolbar = ({
       {onClose && (
         <IconButton
           onClick={onClose}
-          className={cx(classes.iconButton, { 'u-white': !isMobile })}
+          className={cx(classes.iconButton, { 'u-white': isDesktop })}
         >
           <Icon icon={PreviousIcon} />
         </IconButton>
       )}
-      <Typography className="u-pl-half" variant="h3" color="inherit" noWrap>
+      <Typography
+        className="u-pl-half"
+        variant="h3"
+        color={isDesktop ? 'inherit' : 'textPrimary'}
```

Comment:
Why do we need inherit ?

Reply:
pour être blanc et non gris

Reply:
Ah ok on se base sur les couleurs absolues définis dans le stylus, dommage de ne pas utiliser le thème pour ça non ? C'aurait été l'occasion de démarrer le thème dark non ?

Reply:
Oui pour l'instant sur le viewer on est en couleur absolue, on veut spécifiquement du gris sur desktop, sinon du blanc.

Cette PR est un correctif pour sortir une version stable de Drive qui commence à trainer et il y a encore la partie mobile et bottomSheet à traiter. Je n'aurai pas le temps de démarrer le thème dark en plus.